Makefile

```
@@ -2,6 +2,13 @@ SHELL :=/bin/bash
 CWD := $(PWD)
 TMP_PATH := $(CWD)/.tmp
 VENV_PATH := $(CWD)/.venv
+TRAVIS_BRANCH := master
+
+last_tag := $(shell git tag --sort=-creatordate | head -n 1)
+new_tag := $(shell semver bump patch "${last_tag}")
+timestamp := $(date -u +%Y%m%d%H%M%S)
+# The new version is tagged as pre-release for master. Once we are good to go for production, remove the -prerelease suffix
+new_version := $(shell if [ "${TRAVIS_BRANCH}" = "master" ]; then echo "${new_tag}-prerelease"; else echo "${new_tag}-${TRAVIS_BRANCH}.${timestamp}"; fi)
```

Let's call it `alpha` instead of prerelase as it'd already be tagged as pre-release by GitHub.

```suggestion
new_version := $(shell if [ "${TRAVIS_BRANCH}" = "master" ]; then echo "${new_tag}-alpha.${timestamp}"; else echo "${new_tag}-${TRAVIS_BRANCH}.${timestamp}"; fi)
```